consul/tests/common.py

```diff
@@ -36,7 +36,6 @@
     'consul.serf.events',
     'consul.serf.member.join',
     'consul.serf.member.update',
-    'consul.raft.leader.lastContact.count',
```

Comment:
moved to https://github.com/DataDog/integrations-core/pull/9382/files#diff-0417f5405c70f9a67519cb3926358f6100763016af6f781fa24b8e1043bf29a3R123